### PR TITLE
docs(cns): docs content review MTA-4239

### PR DESCRIPTION
**Pull request by SamyOubouaziz**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

**Comment by SamyOubouaziz** [comment]
DO NOT MERGE - will cherry pick in a new PR later

File: console/project/how-to/delete-a-project.mdx [hunk 1]
```diff
@@ -23,23 +23,23 @@ A Project can only be deleted once it has been emptied of its [resources](/ident
...
-      - Click <Icon name="more" /> > **Delete** next to the name of the Project you want to delete.
+    - The default Project cannot be deleted.
     </Message>
-5. Click **Delete Project** to confirm. You are redirected to the default Project dashboard.
+5. Click **Delete Project** to confirm. 
+
+    You are redirected to the default Project dashboard.
+
+<Message type="tip">
+Alternatively, you can delete a project from the **Projects** tab of the [Organization Dashboard](https://console.scaleway.com/organization/projects), by clicking <Icon name="more" />, then **Delete** next to the name of the Project you want to delete.
```
Comment (nerda-codes) [inline console/project/how-to/delete-a-project.mdx:41]:
```suggestion
Alternatively, you can delete a Project from the **Projects** tab of the [Organization Dashboard](https://console.scaleway.com/organization/projects), by clicking <Icon name="more" />, then **Delete** next to the name of the Project you want to delete.
```

File: console/project/how-to/add-resources-project.mdx [hunk 2]
```diff
@@ -22,14 +22,17 @@ To add a new [resource](/identity-and-access-management/iam/concepts/#resource)
   - You have [added and validated a payment method](/console/account/how-to/manage-billing-information/#how-to-add-a-payment-method)
 </Message>
 
-1. Log into the [Scaleway console](https://console.scaleway.com) and click **Project Dashboard** in the side menu to check if you are logged in the correct Project. 
-2. You can also check if the name of your Project appears in the **Selected Project** drop-down menu.
-    <Lightbox src="scaleway-selected-project.webp" alt="" />
-3. Select the resource you want to create from the console side menu.
-4. Click the resource and follow deployment instructions.
+1. Click the **Projects** tab from the [Organization Dashboard](https://console.scaleway.com/organization). A list of your Projects displays.
+2. Click the name of the project to which you want to add resources.
```
Comment (nerda-codes) [inline console/project/how-to/add-resources-project.mdx:26]:
```suggestion
2. Click the name of the Project to which you want to add resources.
```

File: console/project/how-to/add-resources-project.mdx [hunk 3]
```diff
@@ -22,14 +22,17 @@ To add a new [resource](/identity-and-access-management/iam/concepts/#resource)
...
+4. Refer to the corresponding documentation to create the resource.
 
     Once the resource is created, it is automatically included in your Project.
 
+<Message type="tip">
+  Alternatively, you can create other resources via the top-right **Create** button at the top right.
+</Message>
+
 <Message type="note">
   It is currently not possible to transfer resources from one Project to another. For Object Storage, you can [import copies of your objects to buckets in different projects](/storage/object/api-cli/migrating-buckets/).
```
Comment (nerda-codes) [inline console/project/how-to/add-resources-project.mdx:37]:
```suggestion
  It is currently not possible to transfer resources from one Project to another. For Object Storage, you can [import copies of your objects to buckets in different Projects](/storage/object/api-cli/migrating-buckets/).
```

File: console/project/how-to/change-project-name.mdx [hunk 4]
```diff
@@ -21,18 +21,15 @@ categories:
...
-2. Click the **Settings** tab.
-3. Click **Change Project name**.
-    <Lightbox src="scaleway-change-project-name.webp" alt="" />
-
-    A pop-up with a box to edit the name appears.
-    <Lightbox src="scaleway-change-project-name-pop-up.webp" alt="" />
-4. Enter the new name of your Project.
-5. Click **Change Project name**.
+1. Click the **Projects** tab from the [Organization Dashboard](https://console.scaleway.com/organization). A list of your Projects displays.
+2. Click the name of the project you want to edit.
```
Comment (nerda-codes) [inline console/project/how-to/change-project-name.mdx:28]:
```suggestion
2. Click the name of the Project you want to edit.
```

File: console/project/how-to/delete-a-project.mdx [hunk 5]
```diff
@@ -23,23 +23,23 @@ A Project can only be deleted once it has been emptied of its [resources](/ident
   - The Project you want to delete has been emptied of its resources.
 </Message>
 
-1. Click **Project Dashboard** in the [Scaleway console](https://console.scaleway.com) side menu.
+1. Click the **Projects** tab from the [Organization Dashboard](https://console.scaleway.com/organization). A list of your Projects displays.
+2. Click the name of the project you want to delete.
```
Comment (nerda-codes) [inline console/project/how-to/delete-a-project.mdx:27]:
```suggestion
2. Click the name of the Project you want to delete.
```

**Review by bene2k1 — APPROVED** [review]

**Review by nerda-codes — APPROVED** [review]

**Pull request merged**